rpcserverhandlers_test.go

```diff
@@ -636,6 +643,658 @@ func cloneParams(params *chaincfg.Params) *chaincfg.Params {
 	return &paramsCopy
 }
 
+// Block432100 mocks block 432,100 of the block chain.
```

Comment:
What do you think about just grabbing the hex for the block and deserializing it into a memoized block to avoid roughly 700 lines of code here?

You can grab the hex with:
`$ dcrctl getblockhash 432100 | dcrctl getblock - false`


e.g. Something like the following with appropriate comments:

```Go
var blk432100 *wire.MsgBlock
var blk432100Mtx sync.Mutex
func block432100 *wire.MsgBlock {
	defer blk432100Mtx .Unlock()
	blk432100Mtx.Lock()

	if blk432100 != nil {
		return blk432100
	}

	blockBytes, err := hex.DecodeString("hex of block here")
	if err != nil {
		panic(err)
	}

	var block wire.MsgBlock
	err = block.Deserialize(bytes.NewReader(blockBytes))
	if err != nil {
		panic(err)
	}

	blk432100 = &block
	return block432100
}
```

Then you can just get a reference it in the tests via `block := block432100()` and use it the same way.

Reply:
I think that's a good idea. I initially liked this for readability, but I agree that its a ton of extra lines of code. I made the suggested update, but instead of putting the hex inline, I moved it to a test fixture since the hex is still pretty huge.